lib/DDG/Spice/Recipes.pm

```diff
@@ -16,11 +16,11 @@ spice to => 'http://api.yummly.com/v1/api/recipes?q=$1&requirePictures=true&maxR
 handle query_lc => sub {
     return if grep {$req->query_lc eq $_} @stopwords;
 
-    if ($_ =~ s/recipes?//g) {
-        return trim($_);
+    my $ingredient_count = 0;
+    if ($_ =~ s/ ?recipes? ?//g) {
```

Comment:
Consider using `/\s?recipes?\s?/` for the match instead of `/ ?recipes? ?/`. It's virtually the same, but the code is more readable (imho). My modification does allow for tabs, but I don't think that's a deal-breaker - might even be a benefit for weird edge cases (copy/paste for example).